packages/components/organisms/f-checkout/src/components/_tests/AgeVerification.test.js

```diff
@@ -79,6 +79,57 @@ describe('AgeVerification', () => {
                 expect(wrapper.vm.years).toMatchSnapshot();
             });
         });
+
+        describe('isValidAge ::', () => {
+            const validDateOfBirth = new Date(1990, 4, 7);
+            const invalidDateOfBirth = new Date(2021, 4, 7);
+
+            it.each([
+                [true, validDateOfBirth],
+                [false, invalidDateOfBirth]
+            ])('should return %s when dateOfBirth is set to %s', (expectedValue, dateOfBirth) => {
+                // Arrange
+                wrapper = shallowMount(AgeVerification, {
+                    i18n,
+                    localVue,
+                    store: createStore(),
+                    computed: {
+                        userDateOfBirth () {
+                            return dateOfBirth;
+                        }
+                    }
+                });
+
+                // Assert
+                expect(wrapper.vm.isValidAge).toBe(expectedValue);
+            });
+        });
+
+        describe('shouldShowErrorMessage ::', () => {
+            it.each([
+                [true, true, false],
+                [false, true, true],
+                [false, false, true],
+                [false, false, false]
+            ])('should return %s when hasSelectedDateOfBirth is set to %s and isValidAge is %s', (expected, hasSelectedDateOfBirth, isValidAge) => {
+                // Arrange
+                wrapper = shallowMount(AgeVerification, {
+                    i18n,
+                    localVue,
+                    store: createStore(),
+                    computed: {
+                        isValidAge () {
+                            return isValidAge;
+                        }
```

Comment:
You can shorten this using an arrow function.

```suggestion
                        isValidAge: () => isValidAge
```